client/src/pages/OnBoarding/OnBoarding.js

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from "react";
+import { Paper, Grid, Button, makeStyles } from "@material-ui/core";
+import {
+  Route,
+  useHistory,
+  Switch,
+  Redirect,
+  useRouteMatch,
+} from "react-router-dom";
+
+import SetTimezoneUrl from "./SetTimezoneUrl";
+import ConnectGoogleCalendar from "./ConnectGoogleCalendar";
+import SetAvailability from "./SetAvailability";
+
+const OnBoarding = (props) => {
+  //styles
+  const classes = useStyles();
+
+  //data related variables
+  const url = useRef();
+  const timezone = useRef();
+  const days = useRef();
+  const startHour = useRef();
+  const finishHour = useRef();
+  const history = useHistory();
+  const match = useRouteMatch({ path: "/onboarding/:page" });
+  const page = match ? match.params.page : 0;
+
+  const FlipToNextPage = () => {
+    switch (page) {
+      case "1":
+        history.push("/onboarding/2");
+        break;
+      case "2":
+        history.push("/onboarding/3");
```

Comment:
Since we're using `react-router-dom`, let's be consistent and use their components. You're looking for [Link](https://reactrouter.com/web/api/Link), which can be wrapped around a button.
Another less correct option is `Redirect`.